pom.xml

```diff
@@ -35,7 +35,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <ddf.version>2.10.0</ddf.version>
+        <ddf.version>2.11.0-SNAPSHOT</ddf.version>
```

Comment:
Correct. I needed the DDF changes I put up in order for this to work locally. Once 2.10.2 is released, I'll change this. I think I mentioned this in the PR description.